src/Umbraco.Forms.Integrations.Crm.Hubspot/Client/src/lang/en.ts

```diff
@@ -3,5 +3,9 @@ export default {
     formProviderWorkflows:{
         FieldMappingsLabel : `Field Mappings`,
         FieldMappingsDescription: `Map Umbraco Form fields to Hubspot contact fields`,
-    }
+    },
+    hubspotFormWorkflow:{
+        SelectHubspotField : `Select Hubspot field`,
```

Comment:
```suggestion
        SelectHubspotField : `Select HubSpot field`,
```

Please make sure to review for any other cases where we refer to the brand in the UI - it should be `HubSpot`.